pytest/tests/mocknet/load_testing_helper.py

```diff
@@ -62,119 +60,144 @@ def get_latest_block_hash():
     return base58.b58decode(last_block_hash.encode('utf8'))
 
 
-def send_transfer(account, i, i0):
-    next_id = i + 1
-    if next_id >= i0 + NUM_ACCOUNTS:
-        next_id = i0
-    dest_account_id = load_testing_account_id(next_id)
+def send_transfer(account, i, node_account_id):
+    next_id = (i + 1) % mocknet.NUM_ACCOUNTS
+    dest_account_id = mocknet.load_testing_account_id(node_account_id, next_id)
     account.send_transfer_tx(dest_account_id)
 
 
-def random_transaction(account_and_index, i0):
+def random_transaction(account, i, node_account_id):
     choice = random.randint(0, 3)
     if choice == 0:
-        send_transfer(account_and_index[0], account_and_index[1], i0)
+        send_transfer(account, i, node_account_id)
     elif choice == 1:
-        account_and_index[0].send_call_contract_tx('do_work', [])
+        account.send_call_contract_tx('do_work', [])
     elif choice == 2:
-        # min_allowed_top_level_account_length may be set to 32, and accounts creation shorter than it is disabled
         new_account_id = ''.join(
-            random.choice(string.ascii_lowercase) for _ in range(0, 40))
-        account_and_index[0].send_create_account_tx(new_account_id)
+            random.choice(string.ascii_lowercase) for _ in range(0, 10))
+        account.send_create_account_tx(new_account_id)
     elif choice == 3:
-        account_and_index[0].send_stake_tx(1)
+        account.send_stake_tx(1)
 
 
-def send_transfers(i0):
+def send_transfers(node_account_id, test_accounts):
     pmap(
         lambda account_and_index: send_transfer(account_and_index[
-            0], account_and_index[1], i0), test_accounts)
+            0], account_and_index[1], node_account_id), test_accounts)
+
 
+def send_random_transactions(node_account_id, test_accounts):
+    pmap(
+        lambda account_and_index: random_transaction(account_and_index[
+            0], account_and_index[1], node_account_id), test_accounts)
 
-def send_random_transactions(i0):
-    pmap(lambda x: random_transaction(x, i0), test_accounts)
 
+def random_delay():
+    random_delay = random.randint(0, 10)
```

Comment:
Any reason to limit the sleep to integer number of seconds? Perhaps `random_delay = random.random() * 10` instead?